cloudmapper/nodes.py

```diff
@@ -135,7 +135,7 @@ def addChild(self, child):
 
     @property
     def children(self):
-        return self._children.values()
+        return list(self._children.values())
```

Comment:
I used `2to3` for the initial conversion, then back-ported fixes to 2.7.
In this case you are calling `.values()` which returns a list

```python
>>> foo = {'a': 'foo', 'b': 'bar', 'c': 'baz' }
>>> foo.values()
dict_values(['foo', 'bar', 'baz'])
>>> list(foo.values())
['foo', 'bar', 'baz']
```
I am sure that `2to3` made this change in order to be more idiomatic and to avoid a redundant call that will only increase memory.

Reply:
I read your question wrong and deleted my previous comment, 

I used `2to3` for the initial conversion, then back-ported fixes to 2.7.
In this case you are calling `.values()` which returns a list in 2.7 but returns an object of type `dict_values`.  As this is unordered I will remove this wrapper, as keyed values will not be deterministic. 

I will update it with this removed.